example/shipyard.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: "spec.keptn.sh/0.2.2"
+kind: "Shipyard"
+metadata:
+  name: "shipyard-sockshop"
+spec:
+  stages:
+    - name: "production"
```

Comment:
Maybe also add a sequence `delivery` so the user can deploy an image.

Reply:
This is meant as a minimalistic example without delivery.